# inputs
int64 command
---
# outputs
int64 out
